### PR TITLE
Added a tile and an alert too

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/tiel-labyrinth/

**Comment by samyakbvs** [comment]
The Merge branch 'master' into master commits are not being squashed

**Comment by niccokunzmann** [comment]
Please motivate this change and ask other people for review.

**Comment by samyakbvs** [comment]
This change is made so that the labyrinth maze can expand further

**Comment by samyakbvs** [comment]
@niccokunzmann I don't think anyone will review so late in the night so can you merge like this?

File: js/tiles.js [hunk 1]
```diff
@@ -240,11 +253,7 @@ const door = {
     },
     /* Override the function */
     visit: function() {
-      swal({
-        type: 'success',
-        title: 'You win!',
-        text: "Yay! You have won the game"
-      });
+      alert()
```
Comment (niccokunzmann) [inline js/tiles.js]: I do not like this change as it creates an alert box which we removed previously. Why do you remove it?

File: js/tiles.js [hunk 2]
```diff
@@ -273,12 +282,6 @@ const door = {
           this.wallRight = this.createImage("tiles/rooms/wall/rightChess.svg");
           this.ground = this.createImage("tiles/rooms/floor/banner-Chess.svg");
       },
-      visit: function() {
```
Comment (niccokunzmann) [inline js/tiles.js]: You remove this alert box. Although I am for removing alerts, please motivate this, tell us why so the people who added this, understand that the code is better removed.

**Comment by niccokunzmann** [comment]
I do not request changes on the code just on the pull request. Please tell me why.

**Comment by niccokunzmann** [comment]
https://rfc.zeromq.org/spec:42/C4/ - as a reference

**Comment by arjunpandey2003** [comment]
@samyakbvs i agree with @niccokunzmann please don't delete the sweet alert. Rest looks good, also the alert which you have added in your tile please remove it I don't think there is any need for it.

**Comment by samyakbvs** [comment]
Oh sorry I deleted that alert by mistake and okay I will delete my alert

**Comment by arjunpandey2003** [comment]
LGTM it was just that your alert was not necessary you could have given the justification for it.

**Comment by samyakbvs** [comment]
@niccokunzmann pls see

**Comment by niccokunzmann** [comment]
@samyakbvs Thanks for adding the new tile.

In your preview link, the labyrinth does not load. Please correct that, see the console and error messages. Please also take the time to test the changes you provide before you ask other people to do the same.
Here are ideas:
- visit the goal tile to see if it works as before - you modified it
- include your tile into the labyrinth to see if it works.

**Comment by samyakbvs** [comment]
Maaking anew pr

**Pull request closed without merging**